ims/database/image.py

```diff
@@ -141,8 +141,11 @@ def fetch_id_with_name_from_project(self, name, project_name):
             image = self.connection.session.query(Image). \
                 filter(Image.project.has(name=project_name)).filter_by(
                 name=name).one_or_none()
-            if image is not None:
-                return image.id
+
```

Comment:
Some additional blank lines. Please remove them. Also, since this is clean up, adding doc strings, unit tests, copy right terms when we are touching files helps a lot.